examples/step-6/doc/intro.dox

```diff
@@ -3,12 +3,105 @@
 
 @dealiiVideoLecture{15,16,17,17.25,17.5,17.75}
 
-The main emphasis in this example is the handling of locally refined
-grids. The approach to adaptivity chosen in deal.II is to use grids in which
+This program is finally about one of the main features of deal.II:
+the use of adaptively (locally) refined meshes. The program is still
+based on step-4 and step-5, and as you will see, it does not actually
```

Comment:
I believe the comma should be after `and`, i.e., the sentence should read `The program is still based on step-4 and step-5 and, as you will see, it does...` since `as you will see` is a commentary.